src/dsp_tools/commands/project/create/project_create.py

```diff
@@ -1120,17 +1093,62 @@ def create_project(
     # final steps
     if overall_success:
         msg = (
-            f"Successfully created project '{proj_shortname}' ({proj_shortcode}) with all its ontologies. "
+            f"Successfully created project '{project_definition.shortname}' "
+            f"({project_definition.shortcode}) with all its ontologies."
             f"There were no problems during the creation process."
         )
         print(f"========================================================\n{msg}")
         logger.info(msg)
     else:
         msg = (
-            f"The project '{proj_shortname}' ({proj_shortcode}) with its ontologies could be created, "
+            f"The project '{project_definition.shortname}' ({project_definition.shortcode}) "
+            f"with its ontologies could be created, "
             f"but during the creation process, some problems occurred. Please carefully check the console output."
         )
         print(f"========================================================\nWARNING: {msg}")
         logger.warning(msg)
 
     return overall_success
+
+
+def _prepare_and_validate_project(
+    project_json: dict[str, Any],
+) -> ProjectDefinition:
+    project_def = ProjectDefinition(
+        shortcode=project_json["project"]["shortcode"],
+        shortname=project_json["project"]["shortname"],
+        longname=project_json["project"]["longname"],
+        keywords=project_json["project"].get("keywords"),
+        descriptions=project_json["project"].get("descriptions"),
+        groups=project_json["project"].get("groups"),
+        users=project_json["project"].get("users"),
+    )
+
+    # validate against JSON schema
+    validate_project(project_json, expand_lists=False)
+    print("    JSON project file is syntactically correct and passed validation.")
+    logger.info("JSON project file is syntactically correct and passed validation.")
+
+    return project_def
+
+
+def _get_all_lists(project_json: dict[str, Any]) -> list[dict[str, Any]] | None:
+    # expand the Excel files referenced in the "lists" section of the project (if any), and add them to the project
```

Comment:
This comment is not true any more

Reply:
I will do it in the next PR (it is also cleaning)